### ISSUE TITLE
Speaker identification of Link Mate

### ISSUE THREAD

**Issue by thmedia2** [Strixx76/pywam#2]
Hi, 
first of all thanks for the amazing work!
Just one thing I noticed the speaker Identification in the consts.py of the WAM270 (LinkMate) is missing the SPK at the start. At least my identifies as "SPK-WAM270". Would it be possible to change this?

**Comment by Strixx76** [comment]
Yes. I will try to release a new rc next week with improved group handling. I will change this as well for that release.

Please also check [Add support for LinkMate (WAM 270)](https://github.com/Strixx76/samsungwam/issues/5) to help out so that we can get it correct implemented.

**Comment by Strixx76** [comment]
Confirmed working in [v0.1.0rc6](https://github.com/Strixx76/pywam/releases/tag/v0.1.0rc6)
